project_task_code_portal/tests/__init__.py

```
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import test_portal
```

Ruff (0.11.9)

`.test_portal` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)